frontend/public/components/cluster-settings/related-objects.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import * as _ from 'lodash-es';
+import { sortable } from '@patternfly/react-table';
+import { Table, TableRow, TableData, RowFunctionArgs } from '../factory';
+import {
+  referenceForModel,
+  ClusterOperator,
+  ClusterOperatorObjectReference,
+  useModelFinder,
+} from '../../module/k8s';
+import { ResourceLink } from '../utils';
+
+const tableColumnClasses = [
+  '',
+  classNames('pf-m-hidden', 'pf-m-visible-on-sm'),
+  classNames('pf-m-hidden', 'pf-m-visible-on-sm'),
+];
+
+const Header = () => [
+  {
+    title: 'Name',
+    sortField: 'name',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[0] },
+  },
+  {
+    title: 'Kind',
+    sortField: 'resource',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[1] },
+  },
+  {
+    title: 'Namespace',
+    sortField: 'namespace',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[2] },
+  },
+];
+
+const Row: React.FC<RowFunctionArgs> = ({ obj, index, key, style, customData: { findModel } }) => {
+  const { name, resource, namespace, group } = obj;
+  const id = `${index}-${key}`;
+  const model = findModel(group, resource);
+
+  const gsv = model ? referenceForModel(model) : '';
+  return (
+    <TableRow id={id} index={index} trKey={key} style={style}>
+      <TableData className={tableColumnClasses[0]}>
+        <ResourceLink kind={gsv} name={name} namespace={namespace} />
+      </TableData>
+      <TableData className={tableColumnClasses[1]}>{resource}</TableData>
+      <TableData className={tableColumnClasses[2]}>
+        {!_.isEmpty(namespace) ? <ResourceLink kind="Namespace" name={namespace} /> : ''}
+      </TableData>
+    </TableRow>
+  );
+};
+
+const RelatedObjects: React.FC<RelatedObjectsProps> = (props) => {
+  const { findModel } = useModelFinder();
+  return (
+    <div className="co-m-pane__body">
+      <Table
+        {...props}
+        Header={Header}
+        Row={Row}
+        customData={{ findModel }}
+        aria-label="Related Objects"
+      />
+    </div>
+  );
+};
+
+const RelatedObjectsPage: React.FC<RelatedObjectsPageProps> = (props) => {
+  const relatedObject: ClusterOperatorObjectReference[] = props.obj?.status?.relatedObjects;
+  const data = relatedObject.filter(({ name, resource }) => name && resource);
+  return <RelatedObjects {...props} data={data} />;
```

Comment:
What is displayed when a cluster operator has no related objects?

Reply:
Not found is written in middle.
![Screenshot from 2020-06-24 23-34-21](https://user-images.githubusercontent.com/54092533/85609028-48608200-b673-11ea-9320-67eebacf1666.png)

Reply:
> Not found is written in middle.
> ![Screenshot from 2020-06-24 23-34-21](https://user-images.githubusercontent.com/54092533/85609028-48608200-b673-11ea-9320-67eebacf1666.png)

It would be better to say "No Related Objects"

"Not Found" looks like a bug or maybe the ClusterOperator itself is not found.